crates/router/src/compatibility/stripe/customers.rs

```diff
@@ -54,7 +54,7 @@ pub async fn customer_create(
         state.into_inner(),
         &req,
         create_cust_req,
-        |state, auth, req, _| {
+        |state, auth: auth::AuthenticationData, req, _| {
```

Comment:
we do not need this if we feature gate the v2 authentication data

Reply:
Will take this up in a new pr